client/components/auth/loginRegister-modal.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import Modal from 'react-bootstrap/lib/Modal';
+import SigninForm from './signin-form';
+import SignupForm from './signup-form';
+import { Tabs, TabLink, TabContent } from 'react-tabs-redux';
```

Comment:
we are using react-bootstrap already. and maybe we should stick with [react-bootstrap's tab component](https://react-bootstrap.github.io/components.html#tabs) instead of new libraries to reduce the file sizes....is there any reason that we should use this over react-bootstrap's tab?

It's good to see that you are using other open source libraries though!.